sdk/eventhub/event-hubs/test/eventdata.spec.ts

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import chai from "chai";
+chai.should();
+
+import { fromAmqpMessage, toAmqpMessage } from "../src/eventData";
+import { Message } from "rhea-promise";
+
+const testAnnotations = {
+  "x-opt-enqueued-time": Date.now(),
+  "x-opt-offset": "42",
+  "x-opt-sequence-number": 0,
+  "x-opt-partition-key": "key"
+};
+
+const testBody = '{ "foo": "bar" }';
+
+const messageProperties = {
+  message_id: "test_id"
+};
+
+const applicationProperties = {
+  propKey: "propValue"
+};
+
+const testMessage: Message = {
+  body: testBody,
+  message_annotations: testAnnotations,
+  message_id: "test_id",
+  application_properties: applicationProperties
```

Comment:
In track 2, we don't support message_id and message_annotations anymore. So we should remove these from the tests as well as the corresponding changes that are done to `fromAmqpMessage` and `toAmqpMessage` in this PR to support them

Reply:
I separated the source data used for `fromAmqpMessage` and `toAmqpMessage`. The `fromAmqpMessage` tests still ensure that the 4 properties we care about can still be read from `message_annotations`.